ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -45,26 +55,26 @@ protocolInfoByron :: NumCoreNodes
                   -> CoreNodeId
                   -> PBftParams
                   -> Cardano.Genesis.Config
+                  -> Maybe PbftLeaderCredentials
                   -> ProtocolInfo (ByronBlockOrEBB ByronConfig)
-protocolInfoByron (NumCoreNodes numCoreNodes) (CoreNodeId nid) params gc =
+protocolInfoByron (NumCoreNodes numCoreNodes) nid params gc mLeader =
     ProtocolInfo {
         pInfoConfig = WithEBBNodeConfig $ EncNodeConfig {
             encNodeConfigP = PBftNodeConfig {
                   pbftParams   = params
-                    { pbftNumNodes = fromIntegral numCoreNodes
+                    { -- TODO (Duncan): compute (from numGenesisKeys)
+                      pbftNumNodes = fromIntegral numCoreNodes
                       -- Set the signature window to be short for the demo.
+                      -- TODO: Is it always 'k'?
                     , pbftSignatureWindow = 7
+                      -- TODO: make pbftSignatureThreshold default to a value computed from pbftNumNodes,
+                      --       but configurable?
```

Comment:
`pbftSignatureThreshold` is already configurable, it's just not overridden here...

Reply:
The idea, I think, is to supply the configuration from the outside, the same way `PbftLeaderCredentials` got added.